kw.sh

```diff
@@ -5,9 +5,53 @@ EASY_KERNEL_WORKFLOW=${EASY_KERNEL_WORKFLOW:-"kw"}
 src_script_path=${src_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src"}
 external_script_path=${external_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/external"}
 
+# Notification definitions
+declare -r ALERT_NONE=0
+declare -r ALERT_VISUAL=1
+declare -r ALERT_SOUND=2
+declare -r ALERT_BOTH=3
+
 # Export external variables required by kworkflow
 export EASY_KERNEL_WORKFLOW
 
+# Alerts command completion to the user. First argument is the alert code,
+# and second is the command string to be printed in visual notification.
+function alert_completion()
+{
+  local ALERT=$1
+  local COMMAND=$2
+  if [[ $(($ALERT & $ALERT_SOUND)) != 0 ]]; then
+    echo -e "\a\a"
```

Comment:
Replace it for something more flexible. Take a look at my main comment in the PR